### PR TITLE
Portal UI changes

### PR DESCRIPTION
**Pull request by DonaldKLee**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -240,12 +241,15 @@ function App() {
...
             <Sponsors />
           </PageRoute>
-          <PageRoute path="/getting-started">
+          {/* <PageRoute path="/getting-started">
             <GettingStarted />
-          </PageRoute>
+          </PageRoute> */}
           <PageRoute path="/discord-bot">
             <DiscordBot />
           </PageRoute>
```
Comment (meleongg) [inline src/App.js:255]: We prolly want to also comment out this route too if we're not using it

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -209,6 +215,14 @@ export default ({
...
       { location: '/faq', text: 'FAQ' },
     ],
+    useful_links: [
+      {
+        location:
+          'https://nwplus.notion.site/PUBLIC-nwHacks-2024-Hacker-Info-Package-c22183ec8a0f4ccc9900a8200db4fd86',
+        text: 'Hacker Package',
+      },
+      { location: 'https://discord.gg/U3SgBJUHsV', text: 'Discord' },
+    ],
```
Comment (meleongg) [inline src/components/Sidebar.js:225]: Does clicking on the links work for you?

File: src/pages/Home.js [hunk 3]
```diff
@@ -24,21 +27,25 @@ const TopDecal = styled.img`
...
 
-      {/* {user?.status === APPLICATION_STATUS.accepted && isAuthed && user.uid && (
+      {/* Only display QR Code if logged in */}
+      {user?.status === APPLICATION_STATUS.accepted && isAuthed && user.uid ? (
         <QrCode userInfo={user} userId={user.uid} />
-      )} */}
+      ) : (
+        <p style={{ textAlign: 'center' }}>
+          Please login with the account accepted into nwHacks 2024
+        </p>
```
Comment (meleongg) [inline src/pages/Home.js:47]: Might want to use a component from Typography here and change the blurb to say "Please login with the email you used to apply to nwHacks 2024."

File: src/components/Schedule/Timeline.js [hunk 4]
```diff
@@ -77,49 +129,60 @@ const CurrentTime = ({ start, duration, numCols }) => {
...
+              {label === '12:00 AM' && <MobileDayLabel>January 21, 2024</MobileDayLabel>}
+              <br />
+              {label}
+            </MobileTimelineLabel>
+            <br />
+            <MobileTimelineHR hourOffset={i} widthMultiplier={numCols} />
+          </MobileTimelineBlock>
+        )
+      })}
+    </MobileTimelineColumnContainer>
```
Comment (meleongg) [inline src/components/Schedule/Timeline.js:173]: Minor styling & sizing bugs @samanthatuhseng

image

**Comment by meleongg** [comment]
@DonaldKLee login.js page not responsive

- iPhone SE
image

- iPad Air
image

**Comment by meleongg** [comment]
thoughts on making QR code bigger? im scared scanner won't be able to detect? 

image

**Comment by DonaldKLee** [comment]
wait 1 sec, need to fix linting

**Comment by meleongg** [comment]
@DonaldKLee login page still a little wonky on some devices

image
image

nit: can we make the countdown a bit smaller for smaller devices
image

**Comment by samanthatuhseng** [comment]
donalds tickets lgtm

**Comment by samanthatuhseng** [comment]
> @DonaldKLee login page still a little wonky on some devices
> 
> image image
> 
> nit: can we make the countdown a bit smaller for smaller devices image

fixed

**Review by samanthatuhseng — APPROVED** [review]
donalds ticket lgtm

**Pull request merged**